test/support.js

```diff
@@ -19,6 +19,10 @@ expectFileExists = function(file) {
   assert(fs.existsSync(file), file + ' does not exist');
 };
 
+expectFileNotExists = function(file) {
```

Comment:
Since we already have `expectFileExists` (see few lines higher up in this file), I think it's better to keep `expectFileNotExists` for consistency.